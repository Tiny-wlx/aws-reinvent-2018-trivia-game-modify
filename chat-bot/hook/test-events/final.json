{
  "messageVersion": "1.0",
  "invocationSource": "DialogCodeHook",
  "userId": "ozw40htm8dvj0n1lk9ugt6gja961h4dl",
  "sessionAttributes": {
    "started": "true",
    "currentQuestion": "16",
    "currentScore": "0",
    "currentSlot": "sixteen"
  },
  "requestAttributes": null,
  "bot": {
    "name": "TriviaGame",
    "alias": "$LATEST",
    "version": "$LATEST"
  },
  "outputDialogMode": "Text",
  "currentIntent": {
    "name": "LetsPlay",
    "slots": {
      "fifteen": "blah",
      "nine": "blah",
      "six": "blah",
      "one": 2012,
      "seven": "blah",
      "three": "blah",
      "two": "blah",
      "thirteen": "blah",
      "eight": "blah",
      "fourteen": "blah",
      "four": "blah",
      "sixteen": "James Hamilton",
      "twelve": "blah",
      "eleven": "blah",
      "ten": "blah",
      "five": "blah"
    },
    "confirmationStatus": "None"
  },
  "inputTranscript": "Let's play"
}
