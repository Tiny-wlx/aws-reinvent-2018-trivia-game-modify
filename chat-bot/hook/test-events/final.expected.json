{
  "sessionAttributes": {
    "started": "true",
    "currentQuestion": "16",
    "currentScore": "0",
    "currentSlot": "sixteen"
  },
  "dialogAction": {
    "type": "Close",
    "fulfillmentState": "Fulfilled",
    "message": {
      "contentType": "PlainText",
      "content": "That is correct! Thanks for playing! Your final score is 400 points"
    }
  }
}
